src/app/(internal-sites)/chats/[chatId]/page.tsx

```
@@ -279,10 +290,33 @@ export default function Page() {
+      if (!keyPair) {
+        keyPair = await generateKeyPair();
+      }
+
+      if (keyPair && !userInfo.data.publicKey) {
+        const exported = await exportPublicKey(keyPair.publicKey);
+        await updatePublicKey({ publicKey: exported });
+      }
+    }
+
+    void initKeys();
+  }, [userInfo.data, updatePublicKey]);
```

Biome (2.1.2)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)